src/core-taggeds-failure/Failure/Failure.T/Failure.T.Equality.GetHashCode.cs

```diff
@@ -6,11 +6,21 @@ namespace System;
 partial struct Failure<TFailureCode>
 {
     public override int GetHashCode()
```

Comment:
Or it might be implemented in a way like the following.
(But, from my prospective, the first refactor option at https://github.com/pfpack/pfpack-core-taggeds/pull/127/files#r1288853269 is still more preferable for both performance and design purposes).

```cs
    public override int GetHashCode()
    {
        HashCode builder = new();

        builder.Add(EqualityContractHashCode());
        builder.Add(FailureCodeComparer.GetHashCode(FailureCode));
        builder.Add(FailureMessageComparer.GetHashCode(FailureMessage));
        if (SourceException is not null)
        {
            builder.Add(SourceExceptionComparer.GetHashCode(SourceException));
        }

        return builder.ToHashCode();
    }

```

Reply:
Fixed